TransferMethod/Tests/AddTransferMethodPresenterTests.swift

```diff
@@ -1,3 +1,4 @@
+import Common
```

Comment:
still need Common?

Reply:
I think Arun added this because he is testing 
`XCTAssertNotNil(HyperwalletInsights.shared...`

Reply:
No need of that assert then